### PR TITLE
Adding new recipe FGSL.

### PR DESCRIPTION
**Pull request by hhslepicka**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hhslepicka** [comment]
@conda-forge/staged-recipes, this one is ready for a review and merge. Thank you!

File: recipes/fgsl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (wolfv) [inline recipes/fgsl/meta.yaml:1]: maybe remoev the comments here?

File: recipes/fgsl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - {{ compiler('fortran') }}
+    - make
+    - autoconf
+    - automake
+    - m4
+    - pkg-config
+    - libtool
+    - gettext
+  host:
+    - gsl =2.6
```
Comment (wolfv) [inline recipes/fgsl/meta.yaml:32]: can you use either `2.6` or `==2.6`?

File: recipes/fgsl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    - libblas
+    - libcblas
+  run:
+    - gsl =2.6
```
Comment (wolfv) [inline recipes/fgsl/meta.yaml:36]: again

**Comment by wolfv** [comment]
looks good to me!

File: recipes/fgsl/meta.yaml (hunk 1 above)
Comment (hhslepicka) [reply recipes/fgsl/meta.yaml]: Addressed via https://github.com/conda-forge/staged-recipes/pull/16107/commits/962c83fb25a068029f3e64e6794de862fe35a461

File: recipes/fgsl/meta.yaml (hunk 2 above)
Comment (hhslepicka) [reply recipes/fgsl/meta.yaml]: Addressed via https://github.com/conda-forge/staged-recipes/pull/16107/commits/f37be4a5ba7dbfc95d90f3820120946d54149e87

File: recipes/fgsl/meta.yaml (hunk 3 above)
Comment (hhslepicka) [reply recipes/fgsl/meta.yaml]: Addressed via https://github.com/conda-forge/staged-recipes/pull/16107/commits/f37be4a5ba7dbfc95d90f3820120946d54149e87

**Comment by hhslepicka** [comment]
Thanks for the review @wolfv

**Pull request merged**